routes/web.php

```diff
@@ -30,6 +30,11 @@
 })->name('home');
 
 Route::get('/results', function (MismatchGetRequest $request) {
+    if ($request->header('X-Mismatch-Results-Error')) {
+        // force a 500 server response for testing purposes
+        throw new RuntimeException("Simulated Server Error");
+    }
```

Comment:
I'm not sure if we should leave this in the code. But if we do, I'd rather we move this out of the route handler, and maybe create some kind of global middleware for it. In the same way we did for the uselang parameter: https://github.com/wmde/wikidata-mismatch-finder/blob/main/app/Http/Middleware/SetLocale.php

Reply:
moved